.github/workflows/issue-cleanup.yaml

```diff
@@ -20,16 +20,33 @@ jobs:
               owner: context.repo.owner,
```

Comment:
`.github/workflows/issue-cleanup.yaml`, following the convention of the other GitHub Actions would be `.github/workflows/issue-cleanup.yml`.